conf/default.yaml

```diff
@@ -36,3 +36,10 @@ plugins:
         reporter: 'logr-reporter-bell'
         options:
           enabled: false
+  hapi-log-response:
+    tags:
```

Comment:
please remove tags, exclude response and exclude status and we'll just use defaults